### ISSUE TITLE
Pages within the application don't have unique titles

### ISSUE THREAD

**Issue by sandbergja** [pulibrary/lockers_and_study_spaces#196]
### Expected behavior

When I navigate to different pages in the patron or admin interfaces, the `<title>` element changes accordingly, giving me information unique to the page at the beginning. e.g. "Applications awaiting assignment - Lockers and Study Spaces" rather than "Lockers and Study Spaces - Applications awaiting assignment"

### Actual behavior
The `<title>` is always set to `LockerAndStudySpaces`, no matter which page.

### Impact of this bug

[WCAG Level A failure](https://www.w3.org/WAI/WCAG21/Techniques/failures/F25).  Also very annoying for admins and developers who may have many tabs of this application open at once and want to know which is which.
